content/Watsonx/WatsonxData/106.md

```diff
@@ -18,6 +19,8 @@ In this section you will combine data from watsonx.data’s object storage with
 
 2. Click the **Add component** dropdown menu at the right-side of the screen. Select **Add Database**.
 
+<QuizAlert text="Quiz material: pay attention to Add components"/>
```

Comment:
"Add components" -> "the **Add component** menu"

Reply:
changed